config/jobs/kubernetes/minikube/minikube.yaml

```diff
@@ -0,0 +1,28 @@
+presets:
+- env:
+  - name: GCS_CREDS
+    value: /root/.config/k8s-minikube-20e2437ce844.json
```

Comment:
The `mountPath` includes "gcloud", but this does not.

Reply:
Fixed.